### ISSUE TITLE
Shortcut Keys not Advertised to Screen Readers in Palettes

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#28483]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

A small number of Palette items either have default, or can have custom keyboard shortcuts assigned.  For example Layout palette; Insert system break.  It would be helpful if these were advertised to the screen reader. 

STEPS:
1.  With screen reader such as NVDA enabled navigate to the Layout palette and arrow down to System Break.
2. Notice the screen reader just announces System Break and does not read the available shortcut.  It should say "System Break, Ctrl + Enter".

### Supporting files, videos and screenshots

N/A

### What is the latest version of MuseScore Studio where this issue is present?

4.5.2

### Regression

I was unable to check

### Operating system

Windows 11

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by avvvvve** [comment]
Additionally it would be good if tooltips on palette items displayed shortcuts, if they exist (I don't _think_ this happens at the moment)
